Week-2/2.1/src/index.css

```diff
@@ -0,0 +1,63 @@
+header {
```

Comment:
- It would be good to attach screenshot how it is looking like.
- Just to keep in mind, Image size matters in terms of performance, So you could have used optimized size image.